src/app/routes/cpsAsset/index.jsx

```diff
@@ -41,7 +40,7 @@ const CpsAsset = props => {
 export default {
   path: [cpsAssetPagePath, legacyAssetPagePath],
   exact: true,
-  component: memo(CpsAsset, pageIsSame),
+  component: CpsAsset,
```

Comment:
This code was added for live radio too - https://github.com/bbc/simorgh/commit/dffae45ba08ca3bb7dca0c28056d1eb704f0c3c6